dGame/Character.cpp

```diff
@@ -301,20 +301,22 @@ void Character::SaveXMLToDatabase() {
 		character->SetAttribute("gm", m_GMLevel);
 		character->SetAttribute("cc", m_Coins);
 
+		auto zoneInfo = dZoneManager::Instance()->GetZone()->GetZoneID();
 		// lzid garbage, binary concat of zoneID, zoneInstance and zoneClone
-		if (Game::server->GetZoneID() != 0) {
-			auto zoneInfo = dZoneManager::Instance()->GetZone()->GetZoneID();
-			uint64_t lzidConcat = zoneInfo.GetCloneID();
-			lzidConcat = (lzidConcat << 16) | uint16_t(zoneInfo.GetInstanceID());
-			lzidConcat = (lzidConcat << 16) | uint16_t(zoneInfo.GetMapID());
-			character->SetAttribute("lzid", lzidConcat);
-			character->SetAttribute("lnzid", GetLastNonInstanceZoneID());
-
-			//Darwin's backup:
-			character->SetAttribute("lwid", Game::server->GetZoneID());
-
-			// Set the target scene, custom attribute
-			character->SetAttribute("tscene", m_TargetScene.c_str());
+		if ((zoneInfo.GetMapID() != 0) && !(1700 > zoneInfo.GetMapID() && zoneInfo.GetMapID() > 1600)) {
```

Comment:
lup worlds have a should save position flag in the database.  use this instead here.  This comparison is just comparing magic numbers and is not ideal.

Reply:
oh yeah, there's the whole LUPZoneIDs table